.evergreen/scripts/run-aws-tests.sh

```diff
@@ -158,5 +158,42 @@ EOF
   exit
 fi
 
+if [[ "${TESTCASE}" == "ASSUME_ROLE_WITH_WEB_IDENTITY" ]]; then
+  echo "===== Testing auth via Web Identity ====="
+  # Do necessary setup.
+  # Create user on $external db.
+  pushd "${drivers_tools_dir}/.evergreen/auth_aws"
+  mongo --verbose aws_e2e_web_identity.js
+  popd # "${drivers_tools_dir}/.evergreen/auth_aws"
+
+  declare iam_auth_assume_web_role_name iam_web_identity_token_file
+  iam_auth_assume_web_role_name="$(jq -r '.iam_auth_assume_web_role_name' "${drivers_tools_dir}/.evergreen/auth_aws/aws_e2e_setup.json")"
+  iam_web_identity_token_file="$(jq -r '.iam_web_identity_token_file' "${drivers_tools_dir}/.evergreen/auth_aws/aws_e2e_setup.json")"
+
+  echo "Valid credentials via Web Identity - should succeed"
+  AWS_ROLE_ARN="${iam_auth_assume_web_role_name}" \
+  AWS_WEB_IDENTITY_TOKEN_FILE="${iam_web_identity_token_file}" \
+    expect_success "mongodb://localhost/?authMechanism=MONGODB-AWS"
+  echo "Valid credentials via Web Identity with session name - should succeed"
```

Comment:
```suggestion
    expect_success "mongodb://localhost/?authMechanism=MONGODB-AWS"

  echo "Valid credentials via Web Identity with session name - should succeed"
```
Suggest spacing out test blocks for readability.